### DISCUSSION TITLE
How do I target arm64 for net7.0-macos?

### DISCUSSION THREAD

**Discussion by biltongza** [dotnet/runtime#78615] [Q&A] [answered]
Title, when I am targeting net7.0-macos for one of my projects it is building for osx-x64 which is not working on my arm64 machine. Is there something I'm forgetting to set?

**Comment by Fabi** [comment, marked as answer]
> Title, when I am targeting net7.0-macos for one of my projects it is building for osx-x64 which is not working on my arm64 machine. Is there something I'm forgetting to set?

just set the runtime identifier to osx-arm64
